packages/components/.mocharc.js

```diff
@@ -0,0 +1 @@
+module.exports = require("../../presets/react/mocha.config.js");
```

Comment:
`extends: ` didn't work for me. It applied things like `require` but not `reporter` or `ui`

Reply:
probably same for every package, not react specific